en/rocky/8/guides/RL_rsnapshot_backup.md

```diff
@@ -0,0 +1,361 @@
+
+  * Know how to install additional repositories and snapshots from the command-line
+  * Know about mounting filesystems external of your machine (external Hard Drive, remote filesystem, etc.)
+  * Know how to use an editor (vi is used here, but you can use your favorite editor)
+  * Know a little BASH scripting 
+  * Know how to modify the crontab for the root user
+  * Knowledge of SSH public and private keys (only if you plan to run remote backups from another server)
+
+# Installing Rsnapshot
+
+All commands issued here are from the command-line on your server or workstation unless otherwise noted.
+
+## Installing The EPEL
+
+We need the EPEL from Fedora to install the RPM for rsnapshot. This repository should still be available for Rocky Linux to use. To install the repository:
+
+`sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm`
+
+That repository should now be active.
+
+## Install the Rsnapshot Package
+
+Next, install rsnapshot:
+
+`sudo dnf install rsnapshot`
+
+If there are any missing dependencies, those will show up and you simply need to answer the prompt to continue. For example:
+
+```
+dnf install rsnapshot
+Last metadata expiration check: 0:00:16 ago on Mon Feb 22 00:12:45 2021.
+Dependencies resolved.
+========================================================================================================================================
+ Package                           Architecture                 Version                              Repository                    Size
+========================================================================================================================================
+Installing:
+ rsnapshot                         noarch                       1.4.3-1.el8                          epel                         121 k
+Installing dependencies:
+ perl-Lchown                       x86_64                       1.01-14.el8                          epel                          18 k
+ rsync                             x86_64                       3.1.3-9.el8                          baseos                       404 k
+
+Transaction SummaryIn this case, rsnapshot is going to be run locally to backup a particular machine. For this example,  the configuration file will be broken down for specific things that need to be changed in the file.
+========================================================================================================================================
+Install  3 Packages
+
+Total download size: 543 k
+Installed size: 1.2 M
+Is this ok [y/N]: y
+```
+## Mounting A Drive or Filesystem For Backup
+
+This particular step is only necessary if you are backing up a single machine or server (our first example below).
+
+For this particular step, we are assuming an external hard drive of some sort that will be attached to the USB port on your machine, which would likely be the most common method of backing up a single machine. 
```

Comment:
Maybe combine this sentence and the previous one to describe what's going on in this section:
"""
In this step, we show how to mount a hard drive, such as an external USB hard drive, that will be used to back up your system. This particular step is only necessary if you are backing up a single machine or server, as seen in our first example below.
"""